components/actiongroup/index.css

```diff
@@ -53,6 +53,13 @@ governing permissions and limitations under the License.
 .spectrum-ActionGroup--compact {
   gap: var(--mod-actiongroup-gap-size-compact, var(--spectrum-actiongroup-gap-size-compact));
 
+  .spectrum-ActionGroup-item {
```

Comment:
Can this be combined with the definition on line 36?

Reply:
Yeah looks like it can! I felt like there was a reason I kept it separate but I just did it and tested it and it seems to work just fine.